tests/cli/commands/test_internal_api_command.py

```diff
@@ -160,30 +115,37 @@ def test_cli_internal_api_background(self):
                 assert proc.poll() is None
 
                 pid_monitor = self._wait_pidfile(pidfile_monitor)
-                self._wait_pidfile(pidfile_internal_api)
-
+                console.print(f"[blue]Monitor started at {pid_monitor}")
+                pid_internal_api = self._wait_pidfile(pidfile_internal_api)
+                console.print(f"[blue]Internal API started at {pid_internal_api}")
+                console.print("[blue]Running airflow internal-api process:")
                 # Assert that gunicorn and its monitor are launched.
-                assert 0 == subprocess.Popen(["pgrep", "-f", "-c", "airflow internal-api --daemon"]).wait()
+                assert self._find_process(r"airflow internal-api --daemon", print_found_process=True)
```

Comment:
The comment above this state that we are looking for gunicorn and it's monitor, and the next line is looking for `internal-api` processes. (It can be a little bit misleading)

Reply:
Yep. Good catch - it was not what the tests were checking. Corrected the comment.